### DISCUSSION TITLE
Documentation in Portuguese (BR)

### DISCUSSION THREAD

**Discussion by wellwelwel** [sidorares/node-mysql2#2046] [Ideas]
I would like to contribute with the **Portuguese (BR)** documentation like in [简体中文](https://github.com/sidorares/node-mysql2/blob/f930f6e0453f6ffb62f46dec40825f0bff2307a1/documentation/zh-cn/README.md) 📒

<hr />

- [x] Updating documentation path pattern to:
  - documentation
     - { `language`  }
        - `*.md`
  - #2049

<hr />

Then:

- [x] [README.md](https://github.com/sidorares/node-mysql2/blob/f930f6e0453f6ffb62f46dec40825f0bff2307a1/README.md)
  - #2051

<hr />

If you think this is a good idea, I believe it would be really helpful for Brazilians who are starting out in programming or just don't speak English 🇧🇷

**Comment by sidorares** [comment]
What do you think about moving it to under documentation folder?
e.i currently its `documentation` and `documentation_zh-ch`, maybe we can treat english equal to others and have
`documentation/en`, `documentation/zh-ch`, `documentation/pt-br`

**Reply by wellwelwel** [reply]
Yes! This could be done before, in order to already create a new translation with this pattern  defined.

**Comment by sidorares** [comment]
but yeah, this is a great idea and would be great help for those who can't read english well 👍

**Comment by sidorares** [comment]
feel free to propose corrections to original english version if its out of date or can be written better
